sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -470,6 +472,15 @@ public static partial class MultipartResponse
         public static Azure.Response[] Parse(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
         public static System.Threading.Tasks.Task<Azure.Response[]> ParseAsync(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ProcessingContext
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public System.Exception? LastException { get { throw null; } set { } }
+        public System.DateTimeOffset ProcessingStartTime { get { throw null; } set { } }
+        public int RetryNumber { get { throw null; } set { } }
```

Comment:
This is what I had initially, but @KrzysztofCwalina pointed out that it wasn't clear what you are attempting. I guess we could go with RequestAttemptCount, but I think RetryNumber clearly implies that it will be 1 for the first retry.

Reply:
Why not `RetryCount` like @KrzysztofCwalina suggested?

Reply:
I chatted with @KrzysztofCwalina and he was fine with either RetryNumber or RetryCount. I don't have a strong preference.